knora/dsplib/utils/stack_handling.py

```diff
@@ -0,0 +1,67 @@
+import re
+import subprocess
+import time
+
+import requests
+
+from knora.dsplib.models.helpers import BaseError
+
+
+def start_stack() -> None:
+    """
+    Start the Docker containers of DSP-API, and load some basic data models and data.
+    """
+    # start up the fuseki database
+    completed_process = subprocess.run("docker compose up db -d", shell=True, cwd="knora/dsplib/docker")
+    if not completed_process or completed_process.returncode != 0:
+        raise BaseError("Cannot start the API: Error while executing 'docker compose up db -d'")
+    time.sleep(5)
```

Comment:
Why is this sleep needed?

Reply:
@BalduinLandolt and/or @seakayone told me to do it, when we were pair programming. I'm happy to remove it if it is not necessary. What do you think?

Reply:
that was to avoid needing the `wait-for-db` script. Not sure if it's needed.